rest/sql-managed-instance.yml

```diff
@@ -0,0 +1,86 @@
+# Description
+# ===========
+# This playbook create an SQL Managed Instance and all the infrastructure needed
+# As SQL Managed instance is currently available as preview only, it's necessary
+# to have it enabled in selected subscription.
+# Please note that currently it may take more than 24 hours to complete
+
+- name: Create SQL Managed Instance
+  hosts: localhost
+  connection: local
+  vars:
+    resource_group: zimsmanagedthird
+    virtual_network: mivirtualnet
+    route_table: miroutetable
+    subnet: misubnet
+    managed_instance: mibyansible
+    location: eastus
+  tasks:
+
+  - name: Create a resource group
+    azure_rm_resourcegroup:
+      name: "{{ resource_group }}"
+      location: "{{ location }}"
+
+  - name: Create virtual network
+    azure_rm_virtualnetwork:
+      resource_group: "{{ resource_group }}"
+      name: "{{ virtual_network }}"
+      address_prefixes: "10.0.0.0/16"
+
+  - name: Create route table using REST as currently no module is available
```

Comment:
could you pls use route table module?

Reply:
well, after thinking for a while, I think it's ok to leave rest instead of using preview modules right now, as it's under rest samples anyway